### DISCUSSION TITLE
Welcome to qr-code-styling Discussions!

### DISCUSSION THREAD

**Discussion by kozakdenys** [kozakdenys/qr-code-styling#224] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kozakdenys** [comment]
I will start by introducing myself. I’m Denys Kozak, and have been working as a front-end developer for over 9 years. A couple of years ago, after finishing a project and while looking for a new job, I decided to create this library because I saw a lack of good tools for styling QR codes on the front-end.

But then COVID and Russian war in Ukraine destroyed me and I didn't have the energy to make any contribution here and I deciphered from here 😔. But the course of peals and therapy inspired me to quit my job and start looking for things I like to do. So I am here again.

My first steps will be updating dependency vulnerabilities, reviewing open PRs and forks for potential new features, and addressing the most critical issues. I hope you’ll join me on this journey 😊

**Reply by kozakdenys** [reply]
By the way, join me on networks https://linktr.ee/denys.kozak

**Comment by ShahiShohanur** [comment]
I have a simple question. can I change only the first dot color? others mean color 2 and 3 will  remain same.

**Reply by kozakdenys** [reply]
@ShahiShohanur could you please provide some examples?

**Reply by kozakdenys** [reply]
@ShahiShohanur, from what I understand, there is no option to change only one corner color. You can try using JavaScript to get the needed element from SVG and change its attributes. But it is ugly huck 🙃

**Comment by itsouth82** [comment]
Hello
How to add support for Russian Cyrillic, when entering Russian letters it gives an error : "Enter a valid value"

**Comment by SopheakSaing** [comment]
@kozakdenys 
I'm struggling on how should I resolve the missing middle icon of the qr code while downloading on safari?
